packages/calypso-stripe/src/index.tsx

```diff
@@ -272,6 +287,27 @@ export async function confirmStripePaymentIntent(
 	return paymentIntent;
 }
 
+/**
+ * Return Object with mandate option values to be included in creating a SetupIntent
+ *
+ * @param {ShoppingCart} ShoppingCart The shopping cart for the SetupIntent
+ * @returns {Object} An object keyed by the fields needed for the mandate options
+ */
+function getMandateOptionsFromShoppingCart(
+	shoppingCart: ShoppingCart
+): Object {
+	return {
+		amount: 10, // Get value from shopping cart
```

Comment:
We are going to need a function similar to `get_mandate_options_max_amount` which is being introduced in code-D163172. This way, we can make sure the maximum amount take introductory offers and discounts into account.